projects/scion/workbench/src/lib/view-part/view-tab/view-tab.component.ts

```diff
@@ -41,8 +43,14 @@ export class ViewTabComponent {
               private _viewRegistry: WorkbenchViewRegistry,
               private _workbenchLayout: WorkbenchLayoutService,
               private _viewport: SciViewportComponent,
-              private _viewPartService: WorkbenchViewPartService) {
+              private _viewPartService: WorkbenchViewPartService,
+              zone: NgZone) {
     this._host = host.nativeElement;
+    this.installMaximizeListener(zone);
+  }
+
+  ngOnDestroy(): void {
```

Comment:
Please specify `public` access modifier and move this method to the end of the file.

```suggestion
  public ngOnDestroy(): void {
```

Reply:
done